### ISSUE TITLE
Notice headlines always in English

### ISSUE THREAD

**Issue by wahwahbinx** [matcornic/hugo-theme-learn#334]
Hi,

Thank you very much for the theme!

I'm using the [notice shortcodes](https://learn.netlify.com/en/shortcodes/notice/) and have notices that the headlines _Note, Info, Tip_ and _Warning_ are always in English, regardless of the language. You can see that [in your own French guide](https://learn.netlify.com/fr/shortcodes/notice/).

Unfortunately, I haven't managed to manually fix this because I can't seem to find the strings. Preferrably, I'd enhance the translation file in the i18n folder, but even that I haven't managed yet. What do I need to do?

Thanks in advance!

**Comment by Anagastes** [comment]
Same issue here. Have you found a solution?

**Comment by wahwahbinx** [comment]
No, I'm afraid not, I moved to a whole different system for that project.

**Comment by Anagastes** [comment]
May I ask which one? I am still looking for good and fast systems for documentation or FAQ

**Comment by wahwahbinx** [comment]
I went for Typemill, maybe it'll match your use case too.

**Comment by Anagastes** [comment]
For your info, with simple css you can change it. ;) 

In my theme-mine.css:

```
/* multilingual notice */
div.notices.warning:lang(de) p:first-child:after {
    content: 'Warnung';
}
div.notices.note:lang(de) p:first-child:after {
    content: 'Notiz';
}
div.notices.info:lang(de) p:first-child:after {
    content: 'Information';
}
div.notices.tip:lang(de) p:first-child:after {
    content: 'Tipp';
}
```
Here was the solution.
https://github.com/matcornic/hugo-theme-learn/issues/128
